pkg/logs/internal/decoder/auto_multiline_handler.go

```diff
@@ -31,8 +31,13 @@ func NewAutoMultilineHandler(outputFn func(m *message.Message), maxContentSize i
 	}
 
 	return &AutoMultilineHandler{
-		labeler:    automultilinedetection.NewLabeler(heuristics),
-		aggregator: automultilinedetection.NewAggregator(outputFn, maxContentSize, flushTimeout),
+		labeler: automultilinedetection.NewLabeler(heuristics),
+		aggregator: automultilinedetection.NewAggregator(
+			outputFn,
+			maxContentSize,
+			flushTimeout,
+			config.Datadog().GetBool("logs_config.tag_truncated_logs"),
```

Comment:
Ideally we should be using the config component to avoid accessing the global config via `pkg/config`. Not sure how much work would that involve 😄